.bumpversion.cfg

```diff
@@ -8,11 +8,10 @@ tag_name = {new_version}
 search = version = "{current_version}"
 replace = version = "{new_version}"
 
-[bumpversion:file:README.md]
```

Comment:
Removed because the version number in Readme is now being automatically displayed by GitHub badges